Ukoly/DU_6/Piskvorky.py

```diff
@@ -0,0 +1,74 @@
+from random import randrange
+
+WINNER = None
+
+
+def evaluation(string):
+    for _ in string:
+        if 'xxx' in string and 'ooo' not in string:
+            WINNER = '"x"'
+        elif 'ooo' in string and 'xxx' not in string:
+            WINNER = '"o"'
+        elif '-' in string and ('xxx' not in string or 'ooo' not in string):
+            WINNER = None
+        elif '-' not in string and \
+            ('xxx' not in string or 'ooo' not in string):
+            WINNER = 'no one'
+    if WINNER is not None:
+        print('The winner is ' + WINNER)
+    return(WINNER)
+
+
+def move_of_player(field, sign):
+    number_of_field = input('Pick your position. ')
+    while True:
+        number_of_field = str(int(number_of_field) - 1)
+        if not number_of_field.isnumeric():
```

Comment:
Keď si už na riadku vyššie vykonala prevod na číslo a následne si to pretypovala nazad na string tak už to musí byť číslo (int), nakoľko ak by nebolo tak by to spadlo na riadku vyššie (tam mi nejak chýba to overenie)...